modules/frontend/handler.go

```diff
@@ -38,19 +41,21 @@ type (
 // handler exists to wrap a roundtripper with an HTTP handler. It wraps all
 // frontend endpoints and should only contain functionality that is common to all.
 type handler struct {
-	roundTripper http.RoundTripper
-	logger       log.Logger
-	post         handlerPostHook
-	pre          handlerPreHook
+	roundTripper           http.RoundTripper
+	logger                 log.Logger
+	post                   handlerPostHook
+	pre                    handlerPreHook
+	LogQueryRequestHeaders flagext.StringSliceCSV
```

Comment:
This can be lowercase, no need to export it

```suggestion
	logQueryRequestHeaders flagext.StringSliceCSV
```